CHANGELOG.md

```diff
@@ -106,6 +106,12 @@
 - Adds a hint to syntax error when defining a function inside a type.
   ([sobolevn](https://github.com/sobolevn))
 
+- Improve import error messages.
+
+  Now we recommend fixes for `import a.b`
+  and have better suggestions for `import a/b` when `b` is acutally a value.
+  ([sobolevn](https://github.com/sobolevn))
```

Comment:
Could you add an example with a bit of Gleam code, something like this:
```gleam
import gleam.io
```
Showcasing the corresponding error message suggesting the correct import? I think it would be nicer to see examples for people reading the changelog

Reply:
Done!